openhgnn/dataset/NodeClassificationDataset.py

```diff
@@ -528,6 +532,83 @@ def get_split(self, validation=True):
     def get_labels(self):
         return self.label
 
+    def mag4sehgnn(self, dataset):
```

Comment:
Why not directly use the dataset in OpenHGNN? I remember we have implemented the `ogbn-mag` in OpenHGNN.

Reply:
The model HGT preprocessing ogbn-mag in nodeclassificationdataset, which can't be reused for SeHGNN